mod_form.php

```diff
@@ -27,20 +27,22 @@
 
 defined('MOODLE_INTERNAL') || die();
 
-require_once($CFG->dirroot.'/course/moodleform_mod.php');
-require_once($CFG->dirroot.'/lib/questionlib.php');
+require_once($CFG->dirroot . '/course/moodleform_mod.php');
+require_once($CFG->dirroot . '/lib/questionlib.php');
 
 /**
  * Module instance settings form
  * @copyright  2014 John Okely <john@moodle.com>
  * @license    http://www.gnu.org/copyleft/gpl.html GNU GPL v3 or later
  */
-class mod_quizgame_mod_form extends moodleform_mod {
+class mod_quizgame_mod_form extends moodleform_mod
+{
```

Comment:
I need to set up my editor with an autoformatter that follow's moodle's coding style